packages/client/src/exampleTypes/pivot.js

```diff
@@ -212,6 +214,7 @@ export default {
       },
      */
     ],
+    hasResults: false,
```

Comment:
You shouldn't have to set this in the type definition because it's in the node defaults